examples/play_from_file_h264/README.md

```diff
@@ -0,0 +1,29 @@
+# Play from file
+
+This example sends H264 encoded video to a web browser.
+
+## Usage
+
+1. Go to [jsfiddle.net](https://jsfiddle.net/9s10amwL/).
+2. Copy browser SDP offer to file `offer.txt`. 
+**Important** make sure you have copied whole SDP offer, sometimes 
+there might be some `=` characters at the end which might not be selected
+by double-clicking.
+3. Run `run.exs` from the project root directory with:
+
+    ```
+    mix run examples/play_from_file_h264/run.exs (cat examples/play_from_file_h264/offer.txt)
```

Comment:
```suggestion
    mix run examples/play_from_file_h264/run.exs "$(cat examples/play_from_file_h264/offer.txt)"
```

I had to do the following for this to work. I use zsh, but it should also work in bash.

Reply:
Ah, yes, I use fish but let's make this for bash/zsh